src/main/java/frc/robot/Constants.java

```diff
@@ -48,120 +40,30 @@ public static final class DrivetrainConstants {
          */
```

Comment:
Now that PID constants have been moved out of the constants file, do we need this PID tuning comment?